kmultiversx/src/kmultiversx/kdist/mx-semantics/mandos.md

```diff
@@ -25,8 +25,14 @@ module MANDOS
         <newAddresses> .Map </newAddresses>
         <checkedAccounts> .Set </checkedAccounts>
         <elrond/>
+        <txCount> 0 </txCount>
         <exit-code exit=""> 0 </exit-code>
       </mandos>
+    
+    syntax Bytes ::= mkTxHash(Int)    [function, total]
+ // ---------------------------------------------------
+    rule mkTxHash(TX_CNT) => Int2Bytes(32, TX_CNT, BE)
```

Comment:
Just curious: Is this our own implementation of creating a transaction hash, or is this also how Elrond implements it?